blogposts/2021/career-framework.md

```diff
@@ -0,0 +1,314 @@
+---
+title: 'How we built our software engineering career framework'
+description: 'Engineering managers and ICs at Sourcegraph once lacked a common language for growth. Here’s the story of how we developed the first version of our first engineering career framework.'
+author: Nick Moore
+authorUrl: https://twitter.com/nickwritesit
+published: true
+publishDate: 2021-10-07T10:00-07:00
+tags: [blog]
+slug: software-engineer-career-ladder
+heroImage:
+socialImage:
+---
+
+_Sourcegraph provides universal code search: a navigation engine for understanding your code. We’re also an [all-remote](https://about.sourcegraph.com/handbook/company/remote) company with a global [team](https://about.sourcegraph.com/handbook/company/team). See our [careers page](https://boards.greenhouse.io/sourcegraph91) for current openings._
+
+Engineering managers felt empowered but unconfident. It was the summer of 2020 and the company was seven years old and about 50 people strong (about half of whom were engineers). In the year to come, we would grow, rapidly hiring more engineers.
```

Comment:
Since the `description` field from the frontmatter doesn't appear on the blog post itself, I think it'd be worth summarising the post briefly at the beginning so people know what to expect. I'm thinking particularly we may want to allude to this being a very deep dive so they're prepared for a long read 😆